pkgloader-recommended.sty

```diff
@@ -243,3 +243,20 @@
 %    \end{macrocode}
 %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+%  Extracted https://tex.stackexchange.com/a/162436/9075,
+%  where the author of microtype recommends loading settings.
+%  Furthermore, use the hints from https://texdoc.org/pkg/microtype -> "Hints and Caveats"
+%
+%    \begin{macrocode}
+\Load {fontspec} before {microtype}
+                 if {fontspec}
+                 because {the author recommends without any further reasoning}
```

Comment:
I don't think that's enough to warrant inclusion in the default rules.

Reply:
It's not in the docs either. That answer may be dated. The docs do mention `inputenc`.